server/src/lib/api.ts

```diff
@@ -55,20 +56,14 @@ export default class API {
   private getFileContents(filepath: string) {
     return new Promise(
       (res: (contents: string) => void, rej: (error: any) => void) => {
-        fs.readFile(
-          filepath,
-          {
-            contents: 'utf8',
-          },
```

Comment:
Once the typings were imported, TS complained about this parameter. Looking at the available signatures of `readFile`, I guessed that you currently want the one without specifying an encoding (which returns a Buffer).

When you do specify an encoding, `readFile` already gives you a string. We could also change that here.

Since I'm not sure about the effects of this change, and I can't properly test the backend, @mikehenrty please test this thoroughly :)